src/ZIP/src/fmi_miniunz.c

```diff
@@ -62,13 +62,17 @@
 
 #include "unzip.h"
 
+// undefined with minizip-ng
+#ifndef Z_BZIP2ED
+#define Z_BZIP2ED 12
+#endif
```

Comment:
Why is this necessary? From what I can see nothing in this translation unit depends on this define.

Reply:
see line 271:
```
fmi_miniunz.c:271:        if (file_info.compression_method==Z_BZIP2ED)
```